src/ui-kit/InputError/InputError.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+import Icon from 'components/layout/Icon'
+
+import styles from './InputError.module.scss'
+
+
+interface IProps {
```

Comment:
ce que j'ai fait partout dans ma PR infinie : 

```suggestion
interface IInputErrorProps {
```